enterprise/app/filter/date_picker_button.tsx

```diff
@@ -54,17 +54,8 @@ export default class DatePickerButton extends React.Component<Props, State> {
     this.setState({ isOpen: false });
   }
 
-  private onDateChange(range: OnChangeProps) {
-    const selection = (range as { selection: CustomDateRange }).selection;
-    if (selection.days) {
-      router.setQuery({
-        ...Object.fromEntries(this.props.search.entries()),
-        [START_DATE_PARAM_NAME]: "",
-        [END_DATE_PARAM_NAME]: "",
-        [LAST_N_DAYS_PARAM_NAME]: String(selection.days),
-      });
-      return;
-    }
```

Comment:
i'm not sure we really want to remove this--iiuc, this changes "last 7 days" to a specific 7-day date range, so any bookmark set off of this setting will now be referring to a specific date range.

Reply:
good call; I got really confused by the type change here and misunderstood that we would still be getting a CustomDateRange out of `range.selection`. Fixed!